src/blr/utils.py

```diff
@@ -76,3 +77,45 @@ def print_stats(summary, name=None, value_width=15, print_to=sys.stderr):
         print(f"{name:<{max_name_width}} {value:>{value_width},}", file=print_to)
 
     print("="*width, file=print_to)
+
+
+def create_header(file, name, argv):
+    """
+    Create SAM header dict with new tool and command line argument information based on template file. Appends new PG
+    entry with tool name (ID), software name (PN), command line arguments (CL) to track the tools applied to the file.
+    Use in output SAM/BAM file as 'header' attribute.
+
+    :param file: string for SAM/BAM file
+    :param name: string '__name__' variable.
```

Comment:
Do you mean that one should input `__name__` here?

Reply:
Yes!